RescuWise/settings.py

```diff
@@ -27,7 +27,8 @@
 SECRET_KEY = os.environ.get('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+# fix to convert env variable to boolean value
+DEBUG = os.environ['DEBUG'] == "TRUE"
```

Comment:
IMO, best practice dictates that DEBUG should never be an env variable. As in production there is no way to be completely sure that it is set to false unless it is hard coded. I have had a few apps that used env variables for debug and some how debug was defaulting to True. It is also much easier to look at the code base and double check that debug = False visually. I like the intent here, keep it if you want. Just wanted to put my two cents in :D

Reply:
I was going back and forth on this one for sure. I tried my best to thoroughly test to make sure that it consistently worked.

The reason I leaned towards adding it in the env variables was to make it the most user friendly to new developers. I noticed all of us had to add commits fixing the debug value so far hahah. Maybe once we hit a production step we can modify this?

I am totally open to having a discussion about this.